float64[200] jointList
float64[200] commandList
int32 columns
int32 rows
